### ISSUE TITLE
Conda Env Wrapper

### ISSUE THREAD

**Issue by tpowellcio** [conda/conda#104]
We need a wrapper that will make it easier for customers to make the transition from virtualenv to condaenv.  Command suggestions:

$ conda install condawrapper
$ export WORKON_HOME=~/Envs
$ mkdir -p $WORKON_HOME
$ source /usr/local/bin/condawrapper.sh
$ mkcondaenv env1

$ mkcondaenv env1 (creates conda env with default python and conda no additional packages and enter into that environment adding the environment name to the front of the command prompt. i.e.  "(env1) hostname: ~/path $ " )

(env1) $ lssitepackages (lists site packages directory in env)

(env1) $ workon env2 (this is what virtualenv uses to switch between environments we could use the same command? The workon command also includes tab completion for the environment names, and invokes customization scripts as an environment is activated or deactivated)

$ deactivate (Switches from environment to the system-installed version of Python)

$ listcondaenv (lists all conda environments)

Basically, we need to come up with easy-to-use commands similar to http://virtualenvwrapper.readthedocs.org/en/latest/  so existing virtualenv users will make the transition to condaenv with as little effort as possible.

**Comment by mayou36** [comment]
I would love to see that in conda. It's a shame that the activation resp deactivation of an environment needs two commands. I think the virtualenvwrappers behavior could nearly be copied.
Thanks for your efforts in this direction!

**Comment by asmeurer** [comment]
If you activate one environment while another one is already activated, it automatically deactivates first. `deactivate` is only really needed to go back to the root environment (even then, you could `source activate root` instead of `deactivate` if you prefer the command consistency).

**Comment by kalefranz** [comment]
`conda activate` and `conda deactivate` have been implemented in 4.4 as part of #5044 and follow-on PRs.  Hopefully that satisfies the majority of the UX improvement this issues is after.
